Dockerfile

```diff
@@ -27,5 +27,9 @@ COPY --from=ffmpeg /ffmpeg-bin/* /usr/bin/
 EXPOSE 8096
 VOLUME /config /media
 RUN apt-get update \
- && apt-get install -y libfontconfig1 --no-install-recommends  # needed for Skia
+ && apt-get install --no-install-recommends --no-install-suggests -y \
+   libfontconfig1 # Required for Skia \
```

Comment:
Are you sure it's not commenting the rest of the command line? I was bitten by a comment followed by continuation slash in Makefiles already I think

Reply:
Yeah, you're right. The `apt-get install` is the last one that runs for that layer. I'm honestly not clear on how to resolve this without making yet another layer or just plain removing the comment. Suppose I can move the comment between VOLUME and RUN...

Not pretty, but at least then its preserved.